waddrmgr/manager_test.go

```diff
@@ -1906,6 +1931,146 @@ func TestManager(t *testing.T) {
 	}
 }
 
+// TestManagerNewWatchOnly is similar to TestManager, but uses the
+// CreateWatchOnly to test watch-only wallets created from export
```

Comment:
export -> exported

Reply:
Can this be consolidated with `TestManager` by converting to table driven test? Similar to `testImportScript` where there are two test cases with different settings but same execution code.

Reply:
Fixed.